src/Microsoft.DotNet.XHarness.CLI/Commands/WASM/Browser/WasmTestBrowserCommand.cs

```diff
@@ -87,7 +88,29 @@ protected override async Task<ExitCode> InvokeInternal(ILogger logger)
             // getLog() might not see anything for long durations!
             //
             // So -> use a larger timeout!
-            return (driverService, new ChromeDriver(driverService, options, _arguments.Timeout));
+
+            int max_retries = 3;
+            int retry_num = 0;
+            while(true)
+            {
+                try
+                {
+                    return (driverService, new ChromeDriver(driverService, options, _arguments.Timeout));
+                }
+                catch (WebDriverException wde) when (wde.Message.Contains("exited abnormally") && retry_num < max_retries - 1)
```

Comment:
should we make this check for `failed to start: exited abnormally` so we don't accidentally retry other cases of abnormal exits, e.g. if the browser crashes due to some WASM bug while running tests.

Reply:
This is catching exceptions only while starting the browser, before any page gets opened.